spec/requests/api/v2/users/accept_transition_screen_spec.rb

```diff
@@ -60,7 +60,7 @@ def perform_request
       end
 
       it 'returns error' do
-        expect { perform_request }.to raise_error AASM::InvalidTransition
```

Comment:
The spec name no longer matches the spec

Reply:
Thanks i'll change it now.